app/pages/summary/complete/ui-tests/general.ui.test.js

```diff
@@ -116,6 +126,17 @@ test('should render the date completed', async (t) => {
     .expect(await extractInnerText(dateCompleted)).eql(`${content.dateCompletedLabel} ${formattedCurrentDate}`);
 });
 
+test('should render the date completed when on print page', async (t) => {
+  await pageSetup();
+  await t.navigateTo(`${pageUrl}?print=true`);
+
+  const formattedCurrentDate = formatDate(new Date());
+  const dateCompleted = Selector('[data-test-id="date-completed"]');
+
+  await t
+    .expect(await extractInnerText(dateCompleted)).eql(`${content.dateCompletedLabel} ${formattedCurrentDate}`);
```

Comment:
rather than use the currentDate. Can we use the dateCompleted returned from the API via the nock please.